### PR TITLE
remove kinematic points in positivity for nFONLL and add ICasy POS

### PR DESCRIPTION
**Pull request by RoyStegeman**
~~The positivity datasets have observables at x=0.74, which after resealing per Eq. 6 in [Kretzer and Schienbein](https://arxiv.org/pdf/hep-ph/9805233.pdf) means the structure function will be evaluated at chi=0.992. This is a problem since the interplation grids do not contain sufficient points beyond this point.~~

~~I'm not sure the proposed solution is the best but it's what I'm currently using to test the nFONLL implementation. An alternative may simply be to change the xgrid of the postivity observables.~~

~~This PR serves to remind myself of the exact settings I used and to discuss how we may want to address this issue.~~

This PR removes some points

### CODE REVIEW

**Comment by felixhekhorn** [comment]
yeah, we need to find a cleaner solution, this is too sketchy :upside_down_face:

**Comment by RoyStegeman** [comment]
I updated this to remove some points such that the pinecards do not include the points that were excluded in the nFONLL candidate: https://vp.nnpdf.science/kQ3fFdEhSp2tJEeLfCzZ2w==/

It's a bit "ugly" though that the kinematic space covered by the various POS datasets is no longer exactly the same. Thoughts?

**Comment by RoyStegeman** [comment]
@felixhekhorn @giacomomagni @AleCandido what do you think should be done about the kinematic points at which the POS obervables are defined? For nFONLL some points are removed from specific POS datasets, should we remove the same kinematic points from the other POS datasets or are we happy to have some differences between them?

**Comment by AleCandido** [comment]
Actually, as already discussed, it might even have a mild impact (the small-x one), in case the actual difference was still driven by the kinematic threshold.

In general, I fully agree with @giacomomagni

File: NNPDF_POS_F2C_CCP_40/metadata.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+nnpdf_id=POSF2C_CCP
+fktable_id=POSF2C_CCP
+x1_label=Q2
+x1_label_tex=$Q^2$
+x1_unit=GeV^2
+x2_label=x
+x2_label_tex=$x$
+x2_unit=
+y_label=F2c
+y_label_tex=$F_2^{c}$
```
Comment (RoyStegeman) [inline NNPDF_POS_F2C_CCP_40/metadata.txt:14]:
```suggestion
y_label_tex=$F_2^{W^+,c}$
```

File: NNPDF_POS_F2C_CCE_40/metadata.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+nnpdf_id=POSF2C_CCE
+fktable_id=POSF2C_CCE
+x1_label=Q2
+x1_label_tex=$Q^2$
+x1_unit=GeV^2
+x2_label=x
+x2_label_tex=$x$
+x2_unit=
+y_label=F2c
+y_label_tex=$F_2^{c}$
```
Comment (RoyStegeman) [inline NNPDF_POS_F2C_CCE_40/metadata.txt:14]:
```suggestion
y_label_tex=$F_2^{W^-,c}$
```

**Comment by RoyStegeman** [comment]
Given that at the PC last Friday it was agreed to remove these points, can this be merged?

Note that it also adds the positivity needed for ICasy (yes maybe it should've been a separate PR, but oh well)

File: NNPDF_POS_F2C_CCE_40/metadata.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+arxiv=
+description=DIS positivity for charm quark
```
Comment (felixhekhorn) [inline NNPDF_POS_F2C_CCE_40/metadata.txt:2]:
```suggestion
description=positivity for CC DIS anti-charm quark production
```
let's be a bit more specific ...(I hope I got the charge correct :see_no_evil:  )

File: NNPDF_POS_F2C_CCP_40/metadata.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+arxiv=
+description=DIS positivity for charm quark
```
Comment (felixhekhorn) [inline NNPDF_POS_F2C_CCP_40/metadata.txt:2]: see above

**Review by felixhekhorn — COMMENTED** [review]
looks good to me

File: NNPDF_POS_F2C_CCE_40/metadata.txt (hunk 3 above)
Comment (RoyStegeman) [reply NNPDF_POS_F2C_CCE_40/metadata.txt:2]: Just the other way around :)

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**